boat-quay/boat-quay-rules/src/main/kotlin/com/backbase/oss/boat/quay/ruleset/InfoBlockTagsChecker.kt

```diff
@@ -0,0 +1,33 @@
+package com.backbase.oss.boat.quay.ruleset
+
+import com.typesafe.config.Config
+import org.zalando.zally.core.toJsonPointer
+import org.zalando.zally.rule.api.*
+
+@Rule(
+        ruleSet = BoatRuleSet::class,
+        id = "B004",
+        severity = Severity.MUST,
+        title = "Check info block tags allowed."
+)
+class InfoBlockTagsChecker(config: Config) {
```

Comment:
Is there anywhere we can record the justification for the rules? perhaps class kdoc comments?

Reply:
I can add an xdoc. However, I wonder where I can find the justification.

Reply:
@wappers Each rule has an unique ID that will link to a central repostiory holding the description of those rules.

Reply:
Where is this central repostiory?